site/layouts/_default/docs.html

```diff
@@ -1,3 +1,4 @@
+{{ define "body_override" }}<body data-bs-spy="scroll" tabindex="0" data-bs-target="#TableOfContents">{{ end }}
```

Comment:
I think that there should be a block `{{ if (eq .Page.Params.toc true) }}` around to avoid trigger the js when there's no need.

That said, I wonder if these attributes should be on `.bd-main`, `.bd-content` or keep it here ?